internal/plan/plan.go

```diff
@@ -597,6 +677,43 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 		}
 	}
 
+	for name, target := range combined.LogTargets {
+		switch target.Type {
+		case LokiType, SyslogType:
+			// valid, continue
+		default:
+			return nil, &FormatError{
+				Message: fmt.Sprintf(`log target %q has unsupported type %q, must be %q or %q`,
+					name, target.Type, LokiType, SyslogType),
+			}
+		}
+
+		switch target.Selection {
+		case "":
+			// Default (unspecified) selection is opt-out
+			target.Selection = OptOutSelection
```

Comment:
I realize we've done this on a couple of other fields, but I think we should avoid setting values here in `CombineLayers`, otherwise a plan you submit will come back differently than what you submitted (with the default fields filled in). We want to echo back what they submitted. So I think the check for empty string / unset should occur where it's used in the log manager.